### DISCUSSION TITLE
GitHub profile

### DISCUSSION THREAD

**Discussion by SharshenovJR** [SharshenovJR/SharshenovJR#5] [Q&A] [answered]
How to creatively design your GitHub profile to attract attention?

**Comment by SanzharAkparaliev** [comment, marked as answer]
https://www.youtube.com/watch?v=KhGWbt1dAKQ this is wooooorked

**Comment by SanzharAkparaliev** [comment]
https://www.youtube.com/watch?v=rCt9DatF63I also
